### ISSUE TITLE
what file?

### ISSUE THREAD

**Issue by ghost** [maxatwork/form2js#81]
https://github.com/maxatwork/form2js/tree/master/src

I see 1 jquery and 2 .js files.  Which one of the 2 non jquery files should I use?

**Comment by Boycce** [comment]
I agree with @puppetMaster3, the readme should explain each file in the `src` of give clear examples there.
